packages/model-viewer/src/features/scene-graph/sampler.ts

```diff
@@ -15,28 +15,35 @@
 
 import {Texture as ThreeTexture} from 'three';
 
-import {MagFilter, MinFilter, Sampler as GLTFSampler, WrapMode} from '../../three-components/gltf-instance/gltf-2.0.js';
+import {Filter, MagFilter, MinFilter, Sampler as GLTFSampler, Wrap, WrapMode} from '../../three-components/gltf-instance/gltf-2.0.js';
 
 import {Sampler as SamplerInterface} from './api.js';
 import {$correlatedObjects, $onUpdate, $sourceObject, ThreeDOMElement} from './three-dom-element.js';
 
 
 
 const isMinFilter = (() => {
-  const minFilterValues: Array<MinFilter> =
-      [9728, 9729, 9984, 9985, 9986, 9987];
+  const minFilterValues: Array<MinFilter> = [
+    Filter.Nearest,
+    Filter.Linear,
+    Filter.NearestMipmapNearest,
+    Filter.LinearMipmapLinear,
+    Filter.NearestMipmapLinear,
+    Filter.LinearMipmapLinear
+  ];
```

Comment:
Would `= [...Filter]` work here?

Reply:
Interesting, I tried but get
`Type 'typeof Filter' must have a '[Symbol.iterator]()' method that returns an iterator.ts(2488)`
I looked around for enum expansion in typescript but I'm not finding anything off the cuff.